mounts-multistep.yaml

```diff
@@ -0,0 +1,20 @@
+# This template demonstrates mounting custom volumes into a container for multi-step task
+volumes:
+  - name: secret1
+    secret:
+      mysecret1: {{"-----SECRET VALUE 1------" | b64enc}}
```

Comment:
can we use something like  {{ .Values.secret1 | b64enc }}? We don't want to encourage the customers to bake the secret string as plain text in the template =)

Reply:
would I need to provide a corresponding Values file or is it ok to assume they would know how to use that?

Reply:
no need to provide values file